aion/web/elm/src/Room/Notifications.elm

```diff
@@ -17,16 +17,39 @@ addToast toast ( model, cmd ) =
     Toasty.addToast myConfig ToastyMsg toast ( model, cmd )
 
 
+
+-- answer feedback toasts
+
+
 incorrectAnswerToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
 incorrectAnswerToast =
-    addToast (Toasty.Defaults.Error "Error!" "Wrong Answer")
+    addToast (Toasty.Defaults.Error "Error!" "Wrong answer!")
 
 
 closeAnswerToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
 closeAnswerToast =
-    addToast (Toasty.Defaults.Warning "Close one!" "Your Answer is Almost Correct")
+    addToast (Toasty.Defaults.Warning "Close one!" "Your answer is almost correct!")
 
 
 correctAnswerToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
 correctAnswerToast =
-    addToast (Toasty.Defaults.Success "Good Answer!" "Your Answer is Correct")
+    addToast (Toasty.Defaults.Success "Good Answer!" "Your answer is correct!")
+
+
+
+-- question creation
+
+
+questionFormValidationErrorToast : ( Model, Cmd Msg ) -> ( Model, Cmd Msg )
```

Comment:
I feel like we're repeating ourselves a lot with these toasty notifications. 

I think a cooler approach would be to just create three types of Toasty notifications and define error messages in either Errors.elm or Constants.elm (although I would opt for the first one) and just call the function like ErrorToast questionFormValidationError. 

What do you think? We do not need to necessarily introduce it just now as we have big Elm refactor coming anyway.